### ISSUE TITLE
Res101 nuImage config ,please ? 

### ISSUE THREAD

**Issue by Slonna** [HorizonRobotics/Sparse4D#18]
Dear sir, I have tried the bs=4 , num_gpus=4, lr=3e-4, backbone_lr=0.1 ,img_shape=1408x512 , as you proposed in former issue.
However, it seems that it doesn't work (grad_norm will be nan in the training process.
Could you share the Res101 numImg config, I want to reproduce it.
If it is Ok,  my email <slonna119@gmail.com> .
Many thanks!

**Comment by linxuewu** [comment]
I can't send you the company's code; that requires going through a strict company approval process.

Try **pulling the latest code** again, or consider reducing the learning rate.

**Comment by Slonna** [comment]
Ok thanks for  your reply.

**Comment by Slonna** [comment]
Hi, I noticed that last week , one commit update this line ( change the "=" to "copy.deepcopy()"). Is it necessary or just for better view ? ( I think original version "=" is also OK. It seems that  there's no revise on the intrinsic in later code.)
https://github.com/HorizonRobotics/Sparse4D/blob/ea0add848021b8e325e5ac63f51291dccbc46bb0/projects/mmdet3d_plugin/datasets/nuscenes_3d_det_track_dataset.py#L313

**Comment by linxuewu** [comment]
The original code has a bug where cam_intrinsic is modified within the pipeline, resulting in incorrect cam_intrinsic after the first epoch. If you encounter NaN occurring after the first epoch, it may be related to this bug.

> Hi, I noticed that last week , one commit update this line ( change the "=" to "copy.deepcopy()"). Is it necessary or just for better view ? ( I think original version "=" is also OK. It seems that there's no revise on the intrinsic in later code.)
> 
> https://github.com/HorizonRobotics/Sparse4D/blob/ea0add848021b8e325e5ac63f51291dccbc46bb0/projects/mmdet3d_plugin/datasets/nuscenes_3d_det_track_dataset.py#L313

**Comment by sihuaiwei** [comment]
> Dear sir, I have tried the bs=4 , num_gpus=4, lr=3e-4, backbone_lr=0.1 ,img_shape=1408x512 , as you proposed in former issue. However, it seems that it doesn't work (grad_norm will be nan in the training process. Could you share the Res101 numImg config, I want to reproduce it. If it is Ok, my email [slonna119@gmail.com](mailto:slonna119@gmail.com) . Many thanks!

can you send to me the 101 resnet config 295506708@qq.com    I  Used the 101  the errors is :  missing keys in source state_dict:
